### ISSUE TITLE
Config file not loaded when starting service with custom name

### ISSUE THREAD

**Issue by mbaaz** [microsoftarchive/redis#181]
I downloaded the latest binaries from nuget (2.8.17), and tried to install service with a service name and config file. I ran the command:

```
redis-server --service-install --service-name redistest redis.conf
```

and then:

```
redis-server --service-start --service-name redistest
```

After this, when running:

```
redis-cli
> info server
```

the value `config_file` is empty. Also, the default port is used, but the config file specifies a custom port (along with some other values).

It can be said that when running the above without the `--service-name redistest` values, all goes as expected. However, I need to have more than one redis-server installed on my dev machine, so I could really use this feature.

**Comment by lokki** [comment]
I too have this exact issue. Running multiple instances of redis is very common

**Comment by mbaaz** [comment]
My only work-around so far is to add all the settings from config file as console args, that seems to work fine. However, this is no permanent solution.

**Comment by lbukys** [comment]
When run as a service and failing to obey the config file, it's using a default bind setting also!
Note, watching in procmon I see it does open the correct config file (twice), it just seems to ignore all or parts of it.  Manually editing the service's "Path to executable" to remove the --service-name NAME arguments makes the problem go away.  I've stared at the code in BuildServiceRunArguments() which just skips over --service-name NAME in argv and it looks OK, but...

**Comment by jepickett** [comment]
Fixed in commit eb8847498537e4dbffa09d30e2216e37b7baa7e1. Let me know if this works for you.

**Comment by jepickett** [comment]
Closing due to lack of additional discussion.
